pkg/operators/manager.go

```diff
@@ -25,13 +26,23 @@ func RunNucleusOperator(ctx context.Context, controllerContext *controllercmd.Co
 	if err != nil {
 		return err
 	}
-	kubeInformerFactory := informers.NewSharedInformerFactory(kubeClient, 5*time.Minute)
```

Comment:
You will need this eventually.

You probably want to consider a thing like https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/pkg/operator/starter.go#L76-L99